gnmi_server/server_test.go

```diff
@@ -3861,6 +3861,41 @@ print('%s')
 	s.s.Stop()
 }
 
+// Test DPU configuration with multiple databases
```

Comment:
>DPU [](http://example.com/codeflow?start=8&length=3)

This PR is gnmi general support the multi-db feature, not specific related to DPU, but also multi-asic, even single-asic with multiple database. Could you generalize naming/comments/etc ?

Reply:
updated

Reply:
also generalize the PR ttile?